src/org/hammerhead226/sharkmacro/motionprofiles/Profile.java

```diff
@@ -121,8 +124,16 @@ public void execute(int leftGainsProfile, int rightGainsProfile) {
 	 * profile's execution is interrupted.
 	 */
 	public void onInterrupt() {
-		left.onInterrupt();
-		right.onInterrupt();
+		if (left != null) {
+			left.onInterrupt();
+		} else {
+			DriverStation.getInstance().reportError("Tried to interrupt null instance of left ProfileHandler!", false);
```

Comment:
"Tried" is technically not right, maybe something like "No left instance of ProfileHandler to interrupt". Also, probably a warning at strongest, not an error.